benchmarks/tables/nnmf-table.jl

```diff
@@ -3,10 +3,10 @@ include("regulopt-tables.jl")
 Random.seed!(1234)
 m, n, k = 100, 50, 5
 model, A, selected = nnmf_model(m, n, k)
-f = LSR1Model(model)
+f = LBFGSModel(model)
```

Comment:
Why did you change the model here?

Reply:
because, the Bk of LSR1model appears to be non positive hence it gives errors in R2DH